tests/test_cleed_mode.c

```
@@ -0,0 +1,110 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#ifdef _WIN32
+#include <windows.h>
+#else
+#include <unistd.h>
+#endif
+
+#define CLEED_TEST_PATH_SIZE 512
+
+static int write_temp_file(const char *contents, char *path, size_t path_size)
```

Clang (14.0.6)

[warning] parameter 'path_size' is unused

(misc-unused-parameters)